### ISSUE TITLE
Super weird interaction between Liquid filters and the inspect method

### ISSUE THREAD

**Issue by jaredcwhite** [bridgetownrb/bridgetown#72]
Using the console command, if you run `site.render` and then try to run `site.inspect`, you get a weird error and the stack trace seems to indicate a problem in the `inspect` Liquid filter method.

```
Traceback (most recent call last):
       16: from /Users/jared/apps/BRIDGETOWN/bridgetown/bridgetown-core/lib/bridgetown-core/commands/console.rb:58:in `catch'
       15: from /Users/jared/apps/BRIDGETOWN/bridgetown/bridgetown-core/lib/bridgetown-core/commands/console.rb:59:in `block in console'
       14: from (irb):9
       13: from (irb):9:in `rescue in irb_binding'
       12: from (irb):9:in `inspect'
       11: from (irb):9:in `inspect'
       10: from (irb):9:in `inspect'
        9: from (irb):9:in `inspect'
        8: from (irb):9:in `inspect'
        7: from (irb):9:in `inspect'
        6: from (irb):9:in `inspect'
        5: from (irb):9:in `inspect'
        4: from (irb):9:in `inspect'
        3: from (irb):9:in `inspect'
        2: from (irb):9:in `inspect'
        1: from /Users/jared/apps/BRIDGETOWN/bridgetown/bridgetown-core/lib/bridgetown-core/filters.rb:311:in `inspect'
ArgumentError (wrong number of arguments (given 0, expected 1))
```

I cannot for the life of me figure out how that particular method gets mixed into site. It doesn't come up with other objects from what I can tell, nor are there any other filters mixed into site. Spooky!

**Comment by ParamagicDev** [comment]
Based on what I can tell from the error, heres where the issue is occurring.

[https://github.com/bridgetownrb/bridgetown/blob/b884699c004301a0bebd57434db2c9dade43fdda/bridgetown-core/lib/bridgetown-core/filters.rb#L311](https://github.com/bridgetownrb/bridgetown/blob/b884699c004301a0bebd57434db2c9dade43fdda/bridgetown-core/lib/bridgetown-core/filters.rb#L311)

I think whats happening is you have a naming collision.

You are defining an `inspect(arg)` method while simulataneously calling a `xml_escape(input.inspect)` So my best guess would be `input.inspect` is trying to run the `inspect` method you just defined. Let me do some digging...

**Comment by ParamagicDev** [comment]
I tried renaming `inspect` to peek and now it hangs at the error and never exits. Inspect works prior to render. Let's see if something funky is happening in the render method...

**Comment by ParamagicDev** [comment]
Okay, so the issue is definitely with the inspect method listed above.

Remove `inspect(input)` and then do `bundle exec rake install:local`

After this, open up the console.

`bundle exec bridgetown console`

Run:

```bash
site.inspect # no issues
site.render # returns nil
site.inspect # will hang for about 2-3 seconds, then start printing to console
```

**Comment by jaredcwhite** [comment]
Should be fixed in #169. Closing
